### PR TITLE
Connection parameters into dict

### PR DESCRIPTION
**Pull request by KwisatzHaderach**

### CODE REVIEW

**Comment by KwisatzHaderach** [comment]
Should solve Logging #3

File: ovirt_sdk_helpers/connection.py [hunk 1]
```diff
@@ -3,44 +3,43 @@
...
 # Singleton for connection
 con = None
 
-logging.basicConfig(level=logging.DEBUG, filename='base.log')
+logging.basicConfig(
+    level=logging.DEBUG, filename='ovirt_sdk_helper.log'
+)
 logger = logging.getLogger(__name__)
 
+con_params = {
```
Comment (pnovotny) [inline ovirt_sdk_helpers/connection.py:14]: Using `collections.namedtuple` would be even better IMO.

Comment (KwisatzHaderach) [reply ovirt_sdk_helpers/connection.py:14]: discussed offline, wouldn't work as easy as this does

File: ovirt_sdk_helpers/connection.py [hunk 2]
```diff
@@ -3,44 +3,43 @@
...
+    'url': "https://fqdn/ovirt-engine/api",
+    'username': "admin@internal",
+    'password': None,
+    'insecure': True,
```
Comment (pkubica) [inline ovirt_sdk_helpers/connection.py]: Should be 'insecure': False,

File: ovirt_sdk_helpers/connection.py [hunk 3]
```diff
@@ -3,44 +3,43 @@
...
+    'debug': True,
+    'log': logger
+}
+
 
-def init(url_, username_, password_, insecure=True, **kwargs):
+def init(url, username, password, **kwargs):
```
Comment (pkubica) [inline ovirt_sdk_helpers/connection.py:24]: Don't remove insecure parameter, insecure=False

File: ovirt_sdk_helpers/connection.py (hunk 2 above)
Comment (KwisatzHaderach) [reply ovirt_sdk_helpers/connection.py]: will be solved when the issue is fixed by an assignee on that issue :P

File: ovirt_sdk_helpers/connection.py (hunk 3 above)
Comment (KwisatzHaderach) [reply ovirt_sdk_helpers/connection.py:24]: not removed, we have kwargs for the rest now, no need to specify it explicitly

File: ovirt_sdk_helpers/connection.py (hunk 2 above)
Comment (pkubica) [reply ovirt_sdk_helpers/connection.py]: I'm fixing it now and here :smiley:. And now seriously. It still be changed (even if in another pull request) to solve the issue. So don't add new 'bad' code into repository.

File: ovirt_sdk_helpers/connection.py (hunk 3 above)
Comment (pkubica) [reply ovirt_sdk_helpers/connection.py:24]: insecure and ca_file is important arguments that could be visible and defaults could be insecure=False and ca_file="".

**Comment by KwisatzHaderach** [comment]
Should also fix issue #7

**Review by StLuke — APPROVED** [review]

**Review by pkubica — APPROVED** [review]

**Pull request merged**